docs/en/architecture-overview.rst

```diff
@@ -0,0 +1,57 @@
+
+
+The governance level ensures that all participants in the ecosystem are compliant with security and technical standards and requirements. 
+
+.. plantuml:: plantuml/architecture-overview-primary.puml
+    :width: 99%
+    :alt: The image illustrates the IT-Wallet architecture overvew - primary.
+    :caption: `IT-Wallet architecture overview - Primary Actors. <https://www.plantuml.com/plantuml/svg/dPP_Rzky4SV_zwku-HdG18YJFK-I1ocm5lKITpNMBORjh2WmGA2fiqoK8ZNov5Gj-jwdaoYPJjpCUFINVJgUoC_tVl2T9-6ep3LCbwDFGckaP55vmZmPExbHX99AXydCgLouQl0dPQGkKuI_tbvTt5ubjpOlh453RmSVfdUJPB6Sp2Vtoy-poSdfu1NuByeKmeaSj398LO5W7MtiuEFuy_t3ClauURzCvjE_VrWsxi6wl9ybT_UV_jNOxd2j3C9YDfyk9uje0rGM8dMN5zBckJLl3xnPMqCZyOZUvZYwlf3eIAsL58H7UAU2n4fuF2QmpVmHRYfDf73f4PxuLMw4oJ0HcevmVsJT_g_ZXJAU59Mac5PDzhRqP7FmL6dqWt6QJZJcQ6WfifCXVYLqHkZX6VnvbTrQRTsRuI-lcuTj73bO7pbD6Z_hKdJ1cZciXgTFmnPj7kr7_VNNtszMwL5HV_dBwkhcvkfvO4AvsO_wkrnTNgwFYtebRYv_U_qyQcQtJC58t8zzCxvyVVF-kDZNrxzTNe_hs0DIf75NQD0M6iI51YC8fGPZITRv0IlrnFGtMvgq_dyIQ9x1C7Y4aC5nU0hV1r0__G9HqWODAPbuMpg9lbuwxcomQ6t3OBHEaI2rHGZEODVmuD6zyf2gJ97Gm5K2mgGWY9nQbOJrLxyF1X2UlZ-S29-8GjMXSUJHRHN76y-c_B7vPB5SbneMmSwsJxY2tW-_5jPZ2clhGCZQufN9e3zCs5-pcNEO4gYyQ4Nqu4dmqLFr3LCed2KhhVPD96K8dItGYPNIYghc71uboy1DfJHA2guM3u-ADjttWl4CHCleM_EtXL2cYe2SKDgVSoQGoFoREEbz_PlmpJ2kS4wDWxpIrG5vu2uq9gOK_rwqEtTOir9aNGMUh6DGvPvcaLImTZO7SL3h4uUPyhKd-ZC6jsRgrWWDxGjq_hI6lIETEM5ABJg5G5f3Zbap6SRSSj8sotXJP06DBrqhMoD6ve8AZ51Ya2K2oKKUK-nwT4nmZZgvwxAjXTYXOVUmktf5jw3vyMFG_tm5OwqoqwRBLgNeF5W3j47m8aVGk4MzNrPj0quwVqxVKE5MbysfPrqeLehhCbBSa506tScZVxwmyV-dhKdW1PV4k2XsbWRLOsqLHQ7xNEsJdodlciswD98Q0HKfzEUmt605kJ0YmoxhEM7uhrt3NKYMMuSfnnBQdpsh03JfYEm8JGfefAkAlX9AZw70ra8YbrkVVgg9HzstUQU7QgKNkONBWiKwHVHJ6PFVRHYctfSl8kUbFf0-k7HV1pOorBXFBL5kk4zew0A5vjBwdhMdv5MCnU7MVk4_1_06HJQrdyQUbbUP4JmP6Bz9Gvqq_DkkzAnzlmWpt44YcliiNH6pdAEkUBSpuKZXIo1TqODylkwdsVnlT6gjq3NjQm_VbjzKFo45Wg0eLrh91adXr5OGzkCYwahxxRy96f9tDmlEeCbSMrCjDah2LcYLXWHdlOA_tA3yudSVbDLeLOrIP86EJMM-ABwGo2GueLpf2QPcxOGdLqhMlcyVhGBV3umB4abt_jhdtgmidpgatNjCG_lzZs7kSGBltlry_h8YTXVIu2DC33LzX1FISdECMqqVah--gm1gZkPGjvrgeufOAn-jYzM9hkqHvrO8pY-f2j4gJQM7t1hLZhrQYSFpDrOYF8U0ycQJC0JvYhKxtpmOXxrSiULE-J1VK-YMGovKKogCWM59GKBRZ3kjs0gbnKhZuPuRDQM5t60k_Exc4Sye1iAXbOk8CURpBdlu1k0btsYhdRIyAd37549d3YpZUlb3KpXEoYSyXmz5lGvzZwYhYpR6Hr4b3zqKZnhyQVGsxlln1DvtUpQSNN7GBUvyqRvF1p-QA_W_iDhFtFOo6b_TIyCwa31S98EQmFvuwI_0S_nlYPxytfsvyUSRGBo6VIMgBNgGVHh4pljjT-RGew6dployGvEMkVuV>`_
+
+The primary operational layer implements the core Digital Credential lifecycle through coordinated interactions among specialized technical solutions. This layer operates within the trust framework established by governance actors, ensuring that all credential operations maintain security and privacy standards while enabling seamless user experiences.
```

Comment:
```suggestion
The primary operational layer implements the core Digital Credential lifecycle through coordinated interactions among specialized technical solutions, so that all credential operations maintain security and privacy standards while enabling seamless user experiences.
```